client/app/telecom/telephony/alias/special/rsva/telecom-telephony-alias-special-rsva.html

```diff
@@ -0,0 +1,273 @@
+                               data-ng-if="!$ctrl.isEditing"
+                               data-ng-bind="$ctrl.rsvaForm.rateCode.code || '-'">
+                            </p>
+                            <div class="checkbox"
+                                 data-ng-if="$ctrl.isEditing && $ctrl.rateCodeChanged">
+                                <label>
+                                    <input type="checkbox"
+                                           id="editLogged"
+                                           name="editLogged"
+                                           data-ng-model="$ctrl.tariffBearingChangeAgreed" />
+                                    <span data-translate="telephony_alias_special_rsva_tariff_bearing_changed_checkbox"
+                                          data-translate-values="{price: $ctrl.tariffBearingPrice}"></span>
+                                </label>
+                            </div>
+                        </div>
+                        <div class="form-group"
+                             data-ng-if="!$ctrl.isEditing && !$ctrl.rateCodeChanged && $ctrl.rateCodeInfos.pricePerCallWithoutTax.value">
+                            <label class="control-label"
+                                   data-translate="telephony_alias_special_rsva_tariff_bearing_calling_price_label">
+                            </label>
+                            <p class="form-control-static"
+                               data-ng-bind="$ctrl.rateCodeInfos.pricePerCallWithoutTax.value + '€ HT'">
+                            </p>
+                        </div>
+                        <div class="form-group"
+                             data-ng-if="!$ctrl.isEditing && !$ctrl.rateCodeChanged && $ctrl.rateCodeInfos.repaymentPricePerCallWithoutTax.value">
+                            <label class="control-label"
+                                   data-translate="telephony_alias_special_rsva_tariff_bearing_transfer_price_label">
+                            </label>
+                            <p class="form-control-static"
+                               data-ng-bind="$ctrl.rateCodeInfos.repaymentPricePerCallWithoutTax.value + '€ HT'">
+                            </p>
+                        </div>
+                        <div class="row"
+                             data-ng-if="!$ctrl.isEditing">
+                            <div class="oui-tile col-md-10"
+                                 data-ng-if="$ctrl.scheduledRateCode">
+                                <div class="oui-header_5 oui-tile__title"
+                                    data-translate="telephony_alias_special_rsva_scheduled_title"
+                                    data-translate-values="{date: $ctrl.formattedEffectiveDatetime}">
+                                </div>
+                                <div class="oui-tile__body">
+                                    <div class="form-group mx-5">
```

Comment:
Instead of having that structure `.form-group > .control-label + .form-control-static`, you can use regular basic definition list.